.go-version

```diff
@@ -1 +1 @@
-1.20.13
+1.21.6
```

Comment:
this should be 1.22 no?

Reply:
this PR has been around awaiting review for a while :-)

Reply:
we're still on 1.20.13 @ HEAD